src/xdocs/checks/naming/constantname.xml

```diff
@@ -31,7 +31,7 @@
               <td>format</td>
               <td>Specifies valid identifiers.</td>
               <td><a href="../../property_types.html#Pattern">Pattern</a></td>
-              <td><code>"^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$"</code></td>
+              <td><code>&quot;^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$&quot;</code></td>
```

Comment:
The parser escaped those quotes. If I sped more time, I can probably fix it and make it keep them but when I generate the website, those are represented with `"` so it's fine for me.